src/appleseed.studio/mainwindow/project/searchpathswindow.cpp

```diff
@@ -101,7 +101,9 @@ SearchPathsWindow::SearchPathsWindow(
     connect(new QShortcut(QKeySequence(Qt::Key_Delete), this), SIGNAL(activated()), SLOT(slot_remove()));
     connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Up), this), SIGNAL(activated()), SLOT(slot_move_up()));
     connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Down), this), SIGNAL(activated()), SLOT(slot_move_down()));
-
+    connect(new QShortcut(QKeySequence(Qt::CTRL + Qt::Key_Return), this), SIGNAL(activated()), SLOT(close()));
```

Comment:
CTRL+Enter must *accept* the changes that were made, hence the proper slot is `accept()`, not `close()`.